Decks/Source/Deck.cpp

```diff
@@ -1,24 +1,67 @@
 #include "Deck.hpp"
 
+Deck::Deck()
+{
+  cards = std::make_shared<std::vector<Card>>(_cards);
+}
+
+uint Deck::findCard(Card& cardToFind)
+{
+  for(uint i = 0; i < this->cards->size(); ++i)
+  {
+    if(this->cards->at(i) == cardToFind)

```

Comment:
czy dla Card jest zdefiniowany operator==, inaczej to nie będzie działać tak jak chcecie.

Reply:
Jako, że ten std::find jest templatowy to problem chyba zniknął.